server/controllers/user.js

```diff
@@ -61,20 +63,21 @@ export default class UserController {
 
   static login(req, res) {
     const { email, password } = req.body;
-    const found = db.userDb.find(check => check.email === email && check.password === password);
-    if (found) {
-      res.status(200);
-      res.json({
+    const getEmail = email;
+    const getPassword = password;
```

Comment:
why reassign the password? I thought you hashed the password during sign up. Why are you not hashing it during sign in?